bluesky/simple_scans.py

```diff
@@ -61,14 +62,17 @@ def plot_motor(scan):
     return LivePlot(gs.PLOT_Y, scan.motor._name)
 
 
-class _BundledScan:
-    default_subs = []  # subscriptions
-    default_sub_factories = []  # functions that take a scan, return a sub
+class _BundledScan(HasTraits):
```

Comment:
Fixed and pushed

On Sun, Sep 27, 2015 at 5:16 PM Thomas A Caswell <notifications@github.com>
wrote:

> In bluesky/simple_scans.py
> <https://github.com/tacaswell/bluesky/pull/1#discussion_r40509885>:
>
> > @@ -61,14 +62,17 @@ def plot_motor(scan):
> >      return LivePlot(gs.PLOT_Y, scan.motor._name)
> >
> >
> > -class _BundledScan:
> > -    default_subs = []  # subscriptions
> > -    default_sub_factories = []  # functions that take a scan, return a sub
> > +class _BundledScan(HasTraits):
>
> Do not need HasTraits here anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tacaswell/bluesky/pull/1/files#r40509885>.
>